code/game/objects/items/nanoforgedkatana.dm

```diff
@@ -0,0 +1,244 @@
+/obj/item/nanoforgedkatana
+	name = "Nanoforged Katana"
+	desc = "Glorious space nippon steel, folded a million times, producing the finest blade known to mankind. \
+			This blade seems to transmit knowledge of how to use its powers directly to its owner."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "nanoforgedkatana"
+	item_state = "nanoforgedkatana"
+	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
+	flags_1 = CONDUCT_1
+	obj_flags = UNIQUE_RENAME
+	w_class = WEIGHT_CLASS_BULKY
+	force = 30
+	throwforce = 70
+	throw_speed = 3
+	throw_range = 7
+	sharpness = IS_SHARP_ACCURATE
+	embedding = list( "pain_mult" = 0, "jostle_pain_mult" = 0, "embed_chance" = 300, "armour_block" = 100, "fall_chance" = 0, "ignore_throwspeed_threshold" = TRUE)
+	// Last ditch effort to kill someone. Obviously, terrible idea unless the situation is desperate, as it WILL embed and WILL need surgery to remove. No extra embed damage tho, so as to not make it op.
+	hitsound = 'sound/weapons/anime_slash.ogg'
+	armour_penetration = 75
+	block_level = 2
+	block_upgrade_walk = 0 // Base desword blocking, but no possible improvements.
+	block_power = 60 //lower block power.
```

Comment:
this is still insane blocking power.

Reply:
Yeah, I thought it was fairly high, but got told that "yeah well this is shit if it cant block anything !"
What do you think would be good blocking wise ? I still want a way to reach 2 block power so as to fully embrace the animuh "slashing all the bullets"

Reply:
Also, testing wise - 60 block power means 9 hits from esword stam crits the katana.
Which means 2 shotgun blasts.

A desword can't be stamcritted by anything but a shotgun. I tried to stam crit with an esword, but you outheal *naturally* the stam damage. I'll probably lower it to 55, 50, but it's really not as strong as the 70% of desword.